### DISCUSSION TITLE
Extension Files

### DISCUSSION THREAD

**Discussion by cemalgnlts** [CodinGame/monaco-vscode-api#358] [Q&A]
Hi,

Is it possible to access files of the TS Language Server extension from another extension? I am trying to copy this extension: https://github.com/volarjs/vscode-typescript-web/blob/e0c9567fb3fc9e91831eb9764b968f52cd036184/src/client.ts#L32 CDN URL works but cannot access the local TS extension.

It doesn't seem possible from what I have analyzed, but I wanted to ask.

**Comment by CGNonofr** [comment]
Was it closed on purpose?

**Reply by cemalgnlts** [reply]
Yes,
I thought it made more sense to write it in the Discussions section of volar.js.
